lib/map-generator.js

```diff
@@ -202,7 +202,11 @@ class MapGenerator {
     if (this.mapOpts.from) {
       return this.toUrl(this.mapOpts.from)
     } else if (this.mapOpts.absolute) {
-      return pathToFileURL(node.source.input.from).toString()
+      if(pathToFileURL) {
+        return pathToFileURL(node.source.input.from).toString()
+      } else {
+        throw new Error('map option "absolute" is not available in this postcss build')
```

Comment:
Wasn't sure about the error

Reply:
Looks good. Let’s just clean the text:

```suggestion
        throw new Error('`map.absolute` option is not available in this PostCSS build')
```